sci-libs/cantera/cantera-2.4.0.ebuild

```diff
@@ -0,0 +1,144 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6,7}} )
+
+inherit desktop python-r1 scons-utils toolchain-funcs
+
+DESCRIPTION="Object-oriented tool suite for chemical kinetics, thermodynamics, and transport"
+HOMEPAGE="http://www.cantera.org"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+SRC_URI="https://github.com/Cantera/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+IUSE="+cpp +cti fortran pch python static-libs test"
+
+## Python2 is required by '<scons-3.0' to work.
```

Comment:
Then just require new SCons. Adding a lot of hackery to support obsolete build-time deps is not nice.

Reply:
Maybe this comment misleads. Here I just marked that currently portage tree stable `dev-util/scons-2.5.1` or up to `scons-3.0.1-r3` (last seems doesn't support python3 unlike ``scons-3.0.1-r3100`) support only python2 so there is no hard build-time deps on it. 

If user has SCons version that supports python3 then python2 is not necessary to operate SCons build system and to build Cantera python module until user choose to build namely python2 module.

Reply:
Done while doing previous task.